plugins/pulp_rpm/plugins/importers/yum/repomd/presto.py

```diff
@@ -36,7 +37,7 @@ def process_package_element(element):
         oldepoch=delta.attrib['oldepoch'],
         oldversion=delta.attrib['oldversion'],
         oldrelease=delta.attrib['oldrelease'],
-        filename=filename.text,
+        filename=os.path.join("drpms", os.path.basename(filename.text)),
```

Comment:
I see what you're accomplishing here, but it is now unrelated to the rest of this PR. If you want to make a case that this is a worthwhile change, it should be done in a separate PR.